app/assets/stylesheets/govuk_publishing_components/components/_details.scss

```diff
@@ -0,0 +1 @@
+// Use the Details component from GOV.UK Frontend
```

Comment:
Is this file necessary? I get what it's here to explain, but is that important?

Reply:
Yeah, we've done this for every component that uses GOV.UK Frontend. I think it's important so that devs can go to any component's CSS file and discover where the styling comes from. If we didn't have it, it would be up to the developer to find out where the styles come from.